benchmarks/gravity_thin_shell/gravity_thin_shell.prm

```diff
@@ -0,0 +1,82 @@
+# A simple setup for testing the gravity postprocessor using the 
+# sample scheme 'map'.
+
+# General parameters
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output_thin_shell_gravity
+set Nonlinear solver scheme                = no Advection, no Stokes
+
+# Model geometry
```

Comment:
Please add a sentence explaining why there are commented lines in the following section.

Reply:
done